src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -724,7 +725,21 @@ const WorkflowView = _.flow(
               disabled: currentSnapRedacted || !!Utils.computeWorkspaceError(ws),
               checked: useCallCache,
               onChange: v => this.setState({ useCallCache: v })
-            }, [' Use call caching'])
+            }, [' Use call caching']),
+            span({ style: { marginLeft: '1rem', marginRight: '0.5rem' } }, [
+              h(LabeledCheckbox, {
+                checked: deleteIntermediateOutputFiles,
+                onChange: v => this.setState({ deleteIntermediateOutputFiles: v }),
+                style: { marginLeft: '1rem' }
+              }, [' Delete intermediate outputs'])
+            ]),
+            h(InfoBox, [
+              'If the workflow succeeds, only the final output will be saved. Subsequently, call caching cannot be used as the intermediate steps will be not available. ',
```

Comment:
If call caching is mutually exclusive with this option, we should enforce that in the UI somehow, maybe by clearing and disabling the call caching checkbox if you select this option.

Reply:
A workflow can take advantage of previous tasks (calls) whose outputs were cached and also delete it's own intermediate files, so the checkboxes are not mutually exclusive.